transformer_engine/pytorch/ops/fuser.py

```diff
@@ -110,14 +110,6 @@ def forward(
             xs, extra_inputs = _split_tuple(extra_inputs, op.num_extra_inputs)
             basic_op_extra_inputs.append(xs)
 
-        # Get environment state
-        with_quantized_compute = FP8GlobalStateManager.is_fp8_enabled()
-        recipe = FP8GlobalStateManager.get_fp8_recipe() if with_quantized_compute else None
-        is_grad_enabled = func_ctx is not None
-
-        # Attempt to fuse operations if neccesary
-        fuser.maybe_fuse_ops(is_grad_enabled, recipe, input_, basic_op_extra_inputs)
```

Comment:
I've moved the op fusion outside of the autograd context because `torch.is_grad_enabled()` is `False` within autograd functions, which was messing up logic for quantizer usages.